unison-src/transcripts-using-base/fix3166.output.md

```diff
@@ -55,7 +55,7 @@ increment n = 1 + n
            ⧩
            [2, 3, 4]
   
-    22 |     s1 = do emit 10 
```

Comment:
Watch expressions currently naively show only the first line of the expression, so any watch expressions in transcripts which have a 'let' or binding as their first line will be updated since now those are actually considered part of the expression.